test/box-luatest/gh_6305_net_box_autocomplete_test.lua

```diff
@@ -32,18 +32,18 @@ g.test_autocomplete = function()
                         'conn1:call(',
                         'conn1:reload_schema(',
                         'conn1:on_disconnect(',
-                        'conn1:on_shutdown(',
                         'conn1:wait_connected(',
                         'conn1:watch(',
                         'conn1:call_16(',
                         'conn1:execute(',
                         'conn1:wait_state(',
-                        'conn1:ping(',
                         'conn1:unprepare(',
+                        'conn1:ping(',
                         'conn1:prepare(',
                         'conn1:close(',
                         'conn1:on_connect(',
                         'conn1:new_stream(',
+                        'conn1:on_shutdown(',
```

Comment:
Let's change `assert_equals` to `assert_items_equals` so that we don't need to update the test every time we patch net.box connection internals. Here and in all other places in the test. Please consider doing it in a separate commit.